src/views/home/components/deposit-limit-modal/deposit-limit-modal.view.tsx

```diff
@@ -0,0 +1,71 @@
+import { FC } from "react";
```

Comment:
Don't you want to close this modal with ESC or a click in the background as we normally do?

Reply:
Nope, in this case I prefer to avoid that users accidentally click the outside of the modal to close it